### PR TITLE
YTI-2022 term and concept missing properties

### PR DESCRIPTION
**Pull request by kkyttala**
- Add subject area to concept form
- Possibility to add several sources and editorial notes for concept and term

Screenshot 2022-09-05 at 12 23 21

Screenshot 2022-09-05 at 12 27 55

### CODE REVIEW

File: src/common/components/term-modal/index.tsx [hunk 1]
```diff
@@ -172,15 +175,23 @@ export default function TermModal({ data }: TermModalProps) {
...
     }
 
     return (
       <>
         <TermHeading variant="h3">{subtitle}</TermHeading>
-        <TermText>{value}</TermText>
+        {Array.isArray(value) ? (
+          <PropertyList>
+            {value.map((v, idx) => (
+              <li key={`term-source-${idx}`}>{v}</li>
```
Comment (jukkah) [inline src/common/components/term-modal/index.tsx:189]: Key must be unique inside of this map. So no need to use string interpolation. Also index based keys are not considered good practice in React. It's better to use some kind of ID or in this case just `v`. So the result would look like `key={v}`.

File: src/common/components/term-modal/term-expander.tsx [hunk 2]
```diff
@@ -25,7 +29,15 @@ export default function TermExpander({ title, data }: TermExpanderProps) {
             return (
               <div key={`${title}-${idx}`}>
                 <TermHeading variant="h4">{d.subtitle}</TermHeading>
-                <TermText>{d.value}</TermText>
+                {Array.isArray(d.value) ? (
+                  <PropertyList>
+                    {d.value.map((value, idx) => (
+                      <li key={`${value}-${idx}`}>{value}</li>
```
Comment (jukkah) [inline src/common/components/term-modal/term-expander.tsx:35]: Same here.

File: src/modules/concept/administrative-details-expander.tsx [hunk 3]
```diff
@@ -61,10 +62,13 @@ export default function AdministrativeDetailsExpander({
...
           property={concept?.properties.historyNote}
         />
-        <PropertyBlock
-          title={t('field-editorial-note')}
-          property={concept?.properties.editorialNote}
-        />
+        <BasicBlock title={t('field-editorial-note')}>
+          <PropertyList>
+            {concept?.properties.editorialNote?.map((note, idx) => (
+              <li key={`${note}-${idx}`}>{note.value}</li>
```
Comment (jukkah) [inline src/modules/concept/administrative-details-expander.tsx:68]: And here.

File: src/modules/concept/diagrams-and-sources-expander.tsx [hunk 4]
```diff
@@ -41,10 +42,13 @@ export default function DiagramsAndSourcesExpander({
...
           property={undefined}
         />
-        <PropertyBlock
-          title={t('field-sources')}
-          property={concept?.properties.source}
-        />
+        <BasicBlock title={t('field-sources')}>
+          <PropertyList>
+            {concept?.properties.source?.map((source, idx) => (
+              <li key={`${source}-${idx}`}>{source.value}</li>
```
Comment (jukkah) [inline src/modules/concept/diagrams-and-sources-expander.tsx:48]: And here.

**Review by jukkah — CHANGES_REQUESTED** [review]

**Review by tomiviljanen — DISMISSED** [review]

**Review by tomiviljanen — APPROVED** [review]

**Review by jukkah — APPROVED** [review]

**Pull request merged**